---
title: C6103
ms.date: 11/04/2016
ms.topic: reference
f1_keywords:
- C6103
ms.assetid: 22d1ab35-31a3-4ba9-8ef4-7a64bce66621
author: mikeblome
ms.author: mblome
manager: markl
ms.workload:
- multiple
ms.openlocfilehash: 16badf0709e13b2d0710539adf3d8d9e52b08fe6
ms.sourcegitcommit: 485ffaedb1ade71490f11cf05962add1718945cc
ms.translationtype: MT
ms.contentlocale: it-IT
ms.lasthandoff: 10/16/2019
ms.locfileid: "72441907"
---
# <a name="c6103"></a>C6103
avviso C6103: la restituzione di \<variable > dalla chiamata di funzione non riuscita alla riga \<Location >.

 Un percorso corretto tramite la funzione restituisce una variabile utilizzata come parametro `_Out_` a una chiamata di funzione interna non riuscita.

 Il problema potrebbe essere causato dal fatto che la funzione chiamata e la funzione chiamante non sono completamente annotate. La funzione chiamata può richiedere `_Always_`, `_Outptr_result_nullonfailure_` (`_COM_Outptr_` per codice COM) o un'annotazione correlata e la funzione chiamante può richiedere un'annotazione `_Success_`. Un'altra causa possibile del problema consiste nel fatto che l'annotazione `_Success_` nella funzione chiamata non è corretta.

## <a name="see-also"></a>Vedere anche
 [Uso delle annotazioni SAL per ridurre gli errori del codice C/C++](../code-quality/using-sal-annotations-to-reduce-c-cpp-code-defects.md)
